src/app/domains/Plan/components/views/PlanAdvancedView/PlanAdvancedView.template.js

```diff
@@ -1,55 +1,81 @@
 import PropTypes from 'prop-types'
 import { Box } from 'antd-styled'
-import { Typography } from 'antd'
+import { Typography, List, Collapse } from 'antd'
+import { Spinner } from '~/components'
+import { TechnologyAdvancedView } from 'domains/Technology/components/views'
+import { getGrid } from '~/utils'
 import { PlanSimpleView } from 'domains/Plan/components/views'
-import { Status } from '~/components'
-import { FieldTimeOutlined } from '@ant-design/icons'
 import firestore from '~/services/Firebase/firestore/index'
-import { useDocumentData } from 'react-firebase-hooks/firestore'
+import {
+  useCollectionData,
+  useDocumentData
+} from 'react-firebase-hooks/firestore'
 import { COLLECTIONS } from 'app/constants'
-import moment from 'moment'
 const { Text } = Typography
+const { Panel } = Collapse
 /**
  * @info PlanAdvancedView (14 Mar 2021) // CREATION DATE
  *
  * @comment PlanAdvancedView - React component.
  *
- * @since 15 Mar 2021 ( v.0.0.3 ) // LAST-EDIT DATE
+ * @since 16 Mar 2021 ( v.0.0.5 ) // LAST-EDIT DATE
  *
  * @return {ReactComponent}
  */
 
 const PlanAdvancedView = (props) => {
   // [INTERFACES]
-  const { planId } = props
+  const { plan } = props
 
   // [ADDITIONAL_HOOKS]
-  const [plan, loading] = useDocumentData(
-    firestore.collection(COLLECTIONS.PLANS).doc(planId)
+  const [technologies, loadingTechnologies] = useCollectionData(
+    firestore
+      .collection(COLLECTIONS.PLANS)
+      .doc(plan.id)
+      .collection(COLLECTIONS.TECHNOLOGIES)
```

Comment:
You can write that in this way ${COLLECTIONS.PLANS}/${planId}/${COLLECTIONS.TODOS}